modular_framework_core/spawners/manage_object.py

```diff
@@ -121,6 +121,22 @@ def _delete_object(self):
         except rospy.ServiceException as exception:
             rospy.logerr("Delete model service call failed: %s", exception)
 
+
+def triplet_floats_to_list(string_value):
+    """
+        Function parsing a string containing three floats to a list of three floats. For instance "-0.5 0.1 0.4" becomes
+        [-0.5, 0.1, 0.4]
+
+        @param string_value: String containing three elements separated by spaces such as "a b c"
+        @return List corresponding to the input list
+    """
+    print(string_value)
```

Comment:
please remove debug code. If you want to keep this, please add information on what that info is and use rospy.loginfo instead